_posts/2020-10-05-Evidence_locker_overview.md

```diff
@@ -0,0 +1,106 @@
+complete, push changes back to the remote evidence locker.
+
+## Organization
+
+Evidence locker content is namespaced into four top level folders within the
+locker and manages that content on an ongoing basis.  The folders are:
+
+- **raw**: All evidence retrieved by fetchers is stored here.
+- **reports**: All report evidence generated by checks is stored here.
+- **external**: All evidence gathered or generated outside of the Auditree
+framework and planted by [Auditree Plant][] is stored here.
+- **notifications**: All check notifications sent to the locker are stored
+here.
+
+## OK, but what makes it a locker?
+
+...in a word, **metadata**.
+
+As noted earlier, all files are organized according to their data type but
+the Auditree framework still needs a way to treat these files as evidence and
+that's where the metadata comes in.  The top level folders are purposefully
+shallow. Each folder that contains evidence (`notifications` are a special
+case, and not considered evidence) can be further organized with sub-folders
+per category of evidence under `raw`, `reports` and `external`.  Each evidence
+category sub-folder contains an `index.json` file that stores all of the
+metadata for evidences within that category.
+
+Evidence metadata includes:
+
+- Basic evidence metadata
+   - An evidence last update date/time:  You may consider this as a duplication
+   of Git file metadata but this last update date/time is updated as part of
+   every execution of the Auditree framework whereas Git file metadata is only
+   updated when the file content has changed.  This allows us to track the
+   continued execution of fetchers and checks while still leveraging Git's
+   efficient file storage methodology.
+   - An evidence time to live duration
+   - An evidence description
+- Partitioned (raw) evidence metadata
+   - Basic evidence metadata
+   - Fields used to partition the evidence
+   - A mapping of evidence partition to evidence file
+- Report evidence metadata
+   - Basic evidence metadata
+   - Checks used to generate the report
+   - A list of evidence used by the checks to produce the report
+- External evidence metadata
+   - Basic evidence metadata
+   - Planter email
+- Pruned evidence metadata
+   - Basic/Partitioned/Report/External (whichever applies)
+   - Pruner email
+   - Evidence tombstone including an end of life date/time, last update
+   date/time, and a reason for pruning.
+
+In addition to all of the category `index.json` metadata files in the locker,
+you will also find a `check_results.json` in the locker's root.  This file
+contains all of the metadata from the most recent execution of Auditree checks.
+
+Check results metadata is organized by check and contains:
+
+- Accreditations tied a check
+- Check results (successes/failures/warnings)
+- A list of evidence used a check
+- Report(s) generated a check
+
+Finally, an evidence locker has a README.  However this README is special, it
```

Comment:
`README`?

Reply:
Not sure what you mean by this but I made _a_ change in https://github.com/auditree/auditree.github.io/pull/6/commits/b833cf4b4f00db99db6702d9ef909e561d9ce0fc